### ISSUE TITLE
Project Abandoned?

### ISSUE THREAD

**Issue by rndrev** [TravisFSmith/SweetSecurity#49]

**Comment by rndrev** [comment]
Is this project still under active development?

**Comment by zizizach** [comment]
I have the same concern

**Comment by mrbluecoat** [comment]
Subscribe

It would be great if this codebase were refreshed (Bro > Zeek, Critical Stack > Intel Stack, etc)

**Comment by rndrev** [comment]
Yeah I think the only way that would happen is if someone forks it and continues to work on it.  Then they can push the changes back to the master branch if Travis decides to continue his work!
